rest-api/dao/participant_summary_dao.py

```diff
@@ -222,8 +237,11 @@ def update_from_biobank_stored_samples(self, participant_id=None):
                            WHERE biobank_stored_sample.biobank_id = participant_summary.biobank_id
                            AND biobank_stored_sample.test IN %s)
           THEN :received ELSE :unset END
-      ) %s""" % (baseline_tests_sql, dna_tests_sql, sample_sql)
-    params = {'received': int(SampleStatus.RECEIVED), 'unset': int(SampleStatus.UNSET)}
+      ),
+      last_modified = :now
```

Comment:
Add a test that verifies this changes for participants where something changed, and doesn't change for participants where nothing changed, here:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/test/unit_test/offline_test/biobank_samples_pipeline_test.py#L63

Reply:
done